tests/test_config_eq.py

```diff
@@ -99,6 +99,18 @@ def eq(a: Any, b: Any) -> None:
     eq(c2, i2)
 
 
+@mark.parametrize(
+    "cfg,other",
+    [
+        param(DictConfig("???"), "???", id="missing_dictconfig"),
+        param(ListConfig("???"), "???", id="missing_listconfig"),
+    ],
+)
+def test_missing_container_string_eq(cfg: Any, other: Any) -> None:
+    assert cfg == other
+    assert other == cfg
```

Comment:
for completeness, add neq tests.
```suggestion
def test_missing_container_string_eq(cfg: Any, other: Any) -> None:
    assert cfg == other
    assert other == cfg
    assert not (cfg != other)
    assert not (other != cfg)
```